ui/src/app/applications/components/utils.tsx

```diff
@@ -747,7 +747,11 @@ export function syncStatusMessage(app: appModels.Application) {
         if (source.chart) {
             message += ' (' + revision + ')';
         } else if (revision.length >= 7 && !revision.startsWith(source.targetRevision)) {
-            message += ' (' + revision.substr(0, 7) + ')';
+            if (source.repoURL.startsWith('oci://')) {
+                message += ' (' + revision.substring(0, 14) + ')';
```

Comment:
Why 14?

Reply:
This is because ORAS returns a digest prefixed with `sha256:`, so I display that along with the first 7 actual charaters of the digest. I don't have any better way of displaying this atm, suggestions welcome 😄